libs/util/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@cordis/util",
+  "version": "0.1.0",
```

Comment:
fun fact: mono repos aren't versioned this way. even if no changes were made to a package, it should all be lined up.